src/js/components/Select/SelectContainer.js

```diff
@@ -342,12 +351,8 @@ const SelectContainer = forwardRef(
                 type="search"
                 value={search || ''}
                 placeholder={searchPlaceholder}
-                onChange={event => {
-                  const nextSearch = event.target.value;
-                  setSearch(nextSearch);
-                  setActiveIndex(-1);
-                  onSearch(nextSearch);
-                }}
+                onFocus={() => search && executeSearch(search)}
```

Comment:
Why are we updating the search value `onFocus`?

Reply:
Because I guess that is good to allow the dev set some default  `searchValue` and when user click on Select, already filter options based on this default  `searchValue`  by  `onFocus` event.